_candidates_en/YoshitakaOkuda_3.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Yoshitaka Okuda
+title: "Reactive Messaging Patterns for Systems Integrations"
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: https://twitter.com/yoskhdia
+github: https://github.com/yoskhdia
+icon: https://pbs.twimg.com/profile_images/814301502516826112/tkeIfq1H_400x400.jpg
+organization: Supership Inc.
+tags:
+  - Best Practices
+  - Software Design and Architecture
+  - Microservices
+suggestions:
+  - People who want to read Reactive Messaging Patterns
+  - People who are interested in systems integrations using Akka
+  - People who want to know how to apply Akka
+---
+The book "Reactive Messaging Patterns with the Actor Model（Addison-Wesley Professional）"（RMP） was written by Vaughn Vernon, the author of "Implementing Domain-Driven Design（Addison-Wesley Professional）". This book is about how we can implement "Enterprise Integration Patterns"（EIP） using Akka.
+Akka is reactive and resilient. Those characteristics work really well in enterprise systems and systems integrations. As RMP rises, EIP has been established as one of concrete and modern design options. Although, a number of patterns of RMP and EIP is enormous, the information in Japanese is not enough.
```

Comment:
+Akka is reactive and resilient. Those characteristics work really well in enterprise systems and systems integrations. As the use of RMP rises, EIP have been established as concrete and modern design options. Although, a number of patterns of RMP and EIP is enormous, the information in Japanese is not enough.